### PR TITLE
Remove robotpy package (moving to robotpy-cli)

### PR DESCRIPTION
**Pull request by virtuald**

### CODE REVIEW

File: robotpy_meta/__init__.py
Comment (auscompgeek) [inline robotpy_meta/__init__.py:1]: We can have a package with no Python files at all right?

Comment (virtuald) [reply robotpy_meta/__init__.py:1]: I don't think so?

Comment (auscompgeek) [reply robotpy_meta/__init__.py:1]: setuptools seems to be happy to create essentially empty packages

**Comment by auscompgeek** [comment]
TestPyPI accepted this package: https://test.pypi.org/project/robotpy/2024.0.0b4.post1/

**Pull request merged**